### PR TITLE
Improve/groups

### PR DESCRIPTION
**Pull request by zenangst**
- [x] Add tests

### CODE REVIEW

File: Tests/WallTests/Models/GroupSpec.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            expect(group.name).to(equal(name))
+          }
+
+          it("has the correct source URL") {
+            expect(group.image!.thumbnail.string).to(equal(url))
+            expect(group.image!.thumbnail.url).to(beAKindOf(NSURL.classForCoder()))
+          }
+        }
+
+        context("without avatar") {
```
Comment (vadymmarkov) [inline Tests/WallTests/Models/GroupSpec.swift]: Probabably not avatar, but image.

Comment (zenangst) [inline Tests/WallTests/Models/GroupSpec.swift]: @vadymmarkov fixed!

**Comment by vadymmarkov** [comment]
@zenangst Could you fix conflicts?

**Comment by vadymmarkov** [comment]
@zenangst :tada:

**Pull request merged**